test/integration/logic/resolvable/UnifyHasConcludableTest.java

```diff
@@ -436,6 +436,8 @@ public void has_many_to_one_unifier() {
     }
 
     @Test
+    @Ignore
+    //TODO: ignored because this rule is no longer valid
```

Comment:
if its not updatedable to a valid test, we should remove it. Can we fix it here already?

Reply:
I think the test is has to be rewritten. The point is to make a one-to-many unifier, but it's the way it goes about it isn't possible anymore. Therefore I will write a new test to create a one-to-many unifier situation.